Bootstrap.md

```diff
@@ -1,5 +1,30 @@
-A frontend [CSS](https://github.com/bothelpers/kbase/wiki/css) framework.
+# Bootstrap
 
-You can find out more [here](http://getbootstrap.com/)
+### Getting Started
 

```

Comment:
Add a Bootstrap intro and then add a blank header 2 after it

<mini intro to Bootstrap>

```md
## (add a space after ##)
```